### ISSUE TITLE
IF YOU HAVE TROUBLE LOGGING INTO WALLET TRY THIS

### ISSUE THREAD

**Issue by ScotchWiskey** [iotaledger/wallet#704]
Found advise from Korneto02 and it worked for me and lots of others who could not log in. I was so stressed out and now feel like a million $$$. Try this and post if it works for you. I think it likely will.

**hey guys it worked for me go to these site https://iotasalad.org/nodes.... use the nodes there and try all of those, go custom the host and change curl implementation to ccurl...it takes a while like 20-30 minutes says logging in, you just have to wait**

**Comment by CatchTO** [comment]
ScotchWiskey can you please explain this step by step. I have tried the nodes and I could get in to a wallet, but what do I do then, the wallet is empty and the transfer from Bitfinex is no where to see or find. HELP

**Comment by ScotchWiskey** [comment]
OK now that you are logged into your wallet, the advise from IOTA is go to the Receive function of your wallet and generate a new address. Keep generating new addresses until your wallet is updated. That's what I would try since that is what IOTA recommends.

**Comment by CatchTO** [comment]
ok, I will try that thanks

**Comment by ScotchWiskey** [comment]
Yes, I guess the other part to that is you need to keep attaching to tangle, so they you can generate a new address and attach to tangle, generate new address and attach to tangle and keep repeating until your balance is fully updated. Kind of crazy we need to take such steps but apparently a new updated IOTA wallet will be out in a few months and I would think such crazy steps will no longer be necessary with the new wallet.

**Comment by CatchTO** [comment]
I have no tried to generate new addresses with the public node and I have tried to log in with a new seed and transfer the old seed to the new seed and this is now pending. Nothing is happening and I am no afraid that I have make a mistake about generating a new seed and do the transfer from old to new. It a lot of IOTA I have and for me this is a big investment.

**Comment by Em3rgencyLT** [comment]
This worked for me! I've been locked out of my wallet for over a month now. THANK YOU!

**Comment by CatchTO** [comment]
I have no tried to log in with the public node and this went ok, but I am no generating my new address to Tangle for 19 times and nothing happening, do I have to generate more new addresses then this and did you make a new seed before or after you have get your balance back?

**Comment by Em3rgencyLT** [comment]
After you generated your new address, did it show up in the wallet? After I generated 2 or 3 addresses, I noticed none of them were showing up. I then went to both the address and transaction tabs and scrolled down with my mouse, then my wallet stopped responding for about a minute and then everything suddenly loaded in. Full balance and everything :)

**Comment by CatchTO** [comment]
Hmmm. I have no tried to log in 1. with a new generated seed 2. the public node from Germany 3. the seed from my iota transfer from bitfinex(iotasear.ch) I have get in to my our someone's wallet and generated about 20 new addresses but my balance is still 0. Don't understand anything of this, I have my two transferred from bitfinex with addresses and everything on IOTAsear.ch and can see them are completed.
